apps/etterlatte-oppdater-behandling/src/test/kotlin/TidshendelseRiverTest.kt

```diff
@@ -0,0 +1,94 @@
+import io.kotest.matchers.shouldBe
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import no.nav.etterlatte.BehandlingServiceImpl
+import no.nav.etterlatte.TidshendelseRiver
+import no.nav.etterlatte.libs.common.oppgave.OppgaveType
+import no.nav.etterlatte.libs.common.rapidsandrivers.EVENT_NAME_KEY
+import no.nav.etterlatte.rapidsandrivers.ALDERSOVERGANG_ID_KEY
+import no.nav.etterlatte.rapidsandrivers.ALDERSOVERGANG_STEG_KEY
+import no.nav.etterlatte.rapidsandrivers.ALDERSOVERGANG_TYPE_KEY
+import no.nav.etterlatte.rapidsandrivers.DATO_KEY
+import no.nav.etterlatte.rapidsandrivers.DRYRUN
+import no.nav.etterlatte.rapidsandrivers.EventNames
+import no.nav.etterlatte.rapidsandrivers.HENDELSE_DATA_KEY
+import no.nav.etterlatte.rapidsandrivers.SAK_ID_KEY
+import no.nav.helse.rapids_rivers.JsonMessage
+import no.nav.helse.rapids_rivers.testsupport.TestRapid
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import java.time.Month
+import java.time.YearMonth
+import java.util.UUID
+
+class TidshendelseRiverTest {
+    private val behandlingService = mockk<BehandlingServiceImpl>()
+    private val inspector = TestRapid().apply { TidshendelseRiver(this, behandlingService) }
+
+    @Test
+    fun `skal opprette oppgave og returnere dens id`() {
+        val hendelseId = UUID.randomUUID()
+        val sakId = 321L
+        val behandlingsmaaned = YearMonth.of(2024, Month.APRIL)
+        val nyOppgaveID = UUID.randomUUID()
+
+        val melding = lagMeldingForVurdertLoependeYtelse(hendelseId, sakId, behandlingsmaaned, dryRun = false)
+
+        every { behandlingService.opprettOppgave(sakId, behandlingsmaaned, OppgaveType.REVURDERING) } returns nyOppgaveID
+
+        with(inspector.apply { sendTestMessage(melding.toJson()) }.inspektør) {
+            size shouldBe 1
+            field(0, EVENT_NAME_KEY).asText() shouldBe EventNames.ALDERSOVERGANG.name
+            field(0, ALDERSOVERGANG_STEG_KEY).asText() shouldBe "OPPGAVE_OPPRETTET"
+            field(0, ALDERSOVERGANG_TYPE_KEY).asText() shouldBe "BP20"
+            field(0, ALDERSOVERGANG_ID_KEY).asText() shouldBe hendelseId.toString()
+            field(0, DRYRUN).asBoolean() shouldBe false
+            field(0, HENDELSE_DATA_KEY).asText() shouldBe nyOppgaveID.toString()
```

Comment:
Rimelig tungt å lese det her

Reply:
ikke uenig. Nå blir det jo litt uansett pga en del assertions, men en mulig forbedring kan være
```
with (message(0)) {
                this[EVENT_NAME_KEY].asText() shouldBe EventNames.ALDERSOVERGANG.name
                this[ALDERSOVERGANG_STEG_KEY].asText() shouldBe "OPPGAVE_OPPRETTET"
                this[ALDERSOVERGANG_TYPE_KEY].asText() shouldBe "BP20"
                this[ALDERSOVERGANG_ID_KEY].asText() shouldBe hendelseId.toString()
                this[DRYRUN].asBoolean() shouldBe true
                assertThrows<IllegalArgumentException>(message = "Message does not contain field '$HENDELSE_DATA_KEY'") {
                    this[HENDELSE_DATA_KEY]
                }
            }
```

Reply:
Er jo ikke ferdig med det første her, trenger jo f.eks. spesifisere oppgavedetaljer når jeg får dem men merger for å kunne sette i gang test av hele løpet _i dev-miljø_. Så kan man ta litt tid på å diskutere hvordan best asserte slikt som dette